indico/modules/events/templates/display/_event_header_message.html

```diff
@@ -0,0 +1,11 @@
+{% from 'message_box.html' import message_box %}
+
+{% macro render_event_header_msg(event, meeting=true) %}
+    {% set category = event.category %}
+    {% if category.event_message_mode.name != 'disabled' and category.event_message %}
```

Comment:
I believe I'm using `0` for that mode so you could even check `if category.event_message_mode and ..`